components/WhyMeditate/AudioWithSubs.module.css

```diff
@@ -0,0 +1,33 @@
+.word {
+  /*
+      put your CSS here 
+    */
+  animation: appear 0.5s ease-in forwards;
+  opacity: 0;
+}
+
+@keyframes appear {
+  0% {
+    background: #a85a5d;
```

Comment:
I think if you change the color instead of the background, it would look even nicer.